lava_test_plans/include/fastboot.jinja2

```diff
@@ -68,7 +68,6 @@ reboot_to_fastboot: {{ reboot_to_fastboot }}
       minutes: {{ target_deploy_timeout }}
     to: {{ DEPLOY_TARGET }}
     images:
-{% if rootfs == true %}
```

Comment:
Please see:
9ad535d7791a ("fastboot: don't download/flash ptable if rootfs is false")

Reply:
what's your suggestion here for the android case that, no rootfs but needs ptable?

Reply:
whats your way of booting if its not rootfs?
Can't that be used as a flag too? so 'if rootfs == true or xyz == true'... ?

Reply:
with a new variable it works. 
Anyway I just submitted another PR here, please check there if that works,
https://github.com/Linaro/lava-test-plans/pull/250

Reply:
let's discussion at https://github.com/Linaro/lava-test-plans/pull/250 there, it seems we just need to revert the commit https://github.com/Linaro/lava-test-plans/commit/9ad535d7791aa92d180791931bb79fc08309870a ("fastboot: don't download/flash ptable if rootfs is false") now.